### DISCUSSION TITLE
リセットボタンが機能していないので、原因の探究をしたい

### DISCUSSION THREAD

**Discussion by keisuke-yamauch1** [Yowkees/keyball#545] [Q&A] [answered]
## 症状
Pro Microのリセットについてです。

- 単体でGNDとRSTをショートを2回するとリセットされる※ものの、基盤に取り付けてリセットボタンを2回押してもリセットされない。
- 接続時はGNDとRSTをショートさせてもリセットされない

※リセットされるとは、remapに表示されるようになるのでリセットされると表記しました。Pro MicroのLEDが点滅するというのも見ましたが、点灯し続けている状態です

## 推測
上記症状のためはんだ付けを失敗していると考えている
（今回が初めてキーボード作成で、どのように探究すれば良いのかがわかっておりません）

## 質問
こちら（[基盤にmicroproを挿し込むとPC側で認識できなくなる](https://github.com/Yowkees/keyball/discussions/518#discussioncomment-8892730)）を見るに、VCCとGNDの間の電圧を測定すると接続が正いかどうかがわかりそうだと認識しました（リセットボタンとの関連は理解できておりません）が、ここは3.3Vでした。
このあとはどのように原因を探っていけば良いか教えていただけますでしょうか？

また、Pro micro単体ですとGNDとRSTはショートしていませんが、基盤に差し込むとショートしておりました。
これは正しい挙動でしょうか・・・？

**Comment by keisuke-yamauch1** [comment]
色々通電チェックをしていたところ、リセットスイッチ用の2穴がショートしているようでした。
どういった原因が考えられますでしょうか？
お手数ですが写真を添付しましたので、ご確認お願いいたします。

該当箇所の写真
![IMG_6805](https://github.com/Yowkees/keyball/assets/91060865/9463a52f-ee60-4718-8410-846bbd14ea64)

裏全体
![IMG_6806](https://github.com/Yowkees/keyball/assets/91060865/4837bcf7-ef54-4d80-9160-22094545560b)

表全体
![IMG_6807](https://github.com/Yowkees/keyball/assets/91060865/c367b97c-7242-4640-a59b-c6c342f22d71)

**Comment by Yowkees** [comment, marked as answer]
この度はKeyball44をお選びいただき、誠にありがとうございます。

お写真ありがとうございます。
下記写真の緑丸部分がショートしていないでしょうか？ちょうどRSTとGNDの端子です。
ショートしていそうでしたら、半田ごてで加熱して切り離してみてください。

![44Rfix](https://github.com/Yowkees/keyball/assets/84209772/ac232c31-b21c-49fd-b076-d8781f1613ed)

以上、お手数ですがご確認お願いいたします。

**Reply by keisuke-yamauch1** [reply]
@Yowkees 
ご指摘通りショートしておりましたので修正したところ、期待する挙動になりました！
ありがとうございました！！

**Comment by Yowkees** [comment]
無事に修正できたようで安心しました。
ぜひ末永くご愛用いただければ幸いです！
